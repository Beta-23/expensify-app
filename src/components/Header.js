import React from 'react';

// Header stateless functional component for User
const Header = (props) => (
    <div className="header">
        <div className="container">
            <h1 className="header__title">{ props.title }</h1>
            { props.subtitle && <h2 className="header__subtitle">{ props.subtitle }</h2> }
        </div>  
    </div>
);

Header.defaultProps = {
    title: 'iLearnx App'
}

export default Header;