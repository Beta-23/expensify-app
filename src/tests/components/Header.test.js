import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import Header from '../../components/Header';

