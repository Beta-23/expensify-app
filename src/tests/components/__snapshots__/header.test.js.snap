// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    ExpensifyApp
  </h1>
  <[object Object]
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/create"
  >
    Create Expense
  </[object Object]>
  <[object Object]
    activeClassName="is-active"
    to="/help"
  >
    QA
  </[object Object]>
</header>
`;
